specification/communication/data-plane/NetworkTraversal/preview/2021-06-21-preview/CommunicationNetworkTraversal.json

```diff
@@ -26,7 +26,6 @@
           {
             "in": "body",
             "name": "body",
-            "required": true,
```

Comment:
We're expanding the allowed request to not require a field, so without the appropriate service change those requests would fail since the current service API requires a user identity to be given. We need to have this as a new service API version

Reply:
Unless we make the update to the service first to allow these requests to be accepted

Reply:
@jhendrixMSFT do you have a recommendation on if removing a required field requires a new API version or not? And if not, does the service need to be updated to allow the parameter to be optional before making the swagger change in azure-rest-api-specs

Reply:
My understanding is that transitioning from required to not required is not a breaking change (the inverse is).  @johanste is this still the case or did we change the rule?  CI doesn't consider this a breaking change at present but maybe it's outdated.

Reply:
It is not a breaking change, but it is a change that requires a new API version unless explicitly approved - at least for a GA version of the API. In this case, it is a preview API version. @JeffreyRichter, do you have any concerns with this change being made intra-api-version?

Reply:
We like a new version for previews for this kind of change too but this is small enough that it's fine.